src/manage-apps/manage-repos.md

```diff
@@ -0,0 +1,78 @@
+# Managing Repositories
```

Comment:
This page does not talk about managing the built-in repositories (particularly, the Official Sources section of the Settings tab in Repoman.) It's not necessarily required that this is covered (it's usually not necessary to change, especially since we switched to the CDN setup), but if I wanted someone to understand how their system worked/where their software came from, that would be a pretty basic part of it.

I do notice there are a couple of quirks with Repoman (the `Reset Mirrors to Defaults` button is outdated and points back to Canonical's Ubuntu server, and the Extra Sources page says the sources are third-party/may present a security risk even though that page includes two official Pop!_OS sources by default), so if it's easier to skip it for now, that's fine.

One other thing not covered is managing repositories manually using text files and/or the CLI, which could be a separate page if we wanted to cover that. Pop!_OS has a unique `apt-manage` command (it comes from repolib, which is what repoman is built on top of) for working with deb repos. The text files themselves are located in `/etc/apt/sources.list.d/` and `~/.local/share/flatpak/repo/`.

Reply:
I'll re-write this section to include Official Sources. I'll also create a new section for apt-manage. Section reorganization will look like this: 

- Manage Repositories (title section, describes methods, background info, use-cases)
- - Using Repoman (GUI) 
- - Using apt-manage (CLI)